autograder/api/config.py

```diff
@@ -379,6 +379,15 @@ def __call__(self, parser, namespace, raw_values, option_string = None):
         'nargs': '+',
     })
 
+PARAM_REGRADE_AFTER = APIParam('regrade-after',
+    ('The timestamp that will be applied to the request.'
+    + ' Every target user will have a new submission after this time.'
+    + ' By default, the current time will be used.'
+    + ' Use this option to manually set the regrade time.'
```

Comment:
Your language here is pretty confusing.
I would keep it simple, like "All submission occurring before this time will be regraded.".

Would it make more sense to rename this option "regrade-before" or "regrade-cutoff"?